WordPress/Classes/ViewRelated/Post/PostActionSheet.swift

```diff
@@ -107,6 +111,7 @@ class PostActionSheet {
         static let retry = NSLocalizedString("Retry", comment: "Retry uploading the post.")
         static let edit = NSLocalizedString("Edit", comment: "Edit the post.")
         static let share = NSLocalizedString("Share", comment: "Share the post.")
+        static let blaze = NSLocalizedString("Promote with Blaze", comment: "Promote the post with Blaze.")
```

Comment:
[Suggestion] Using reverse DNS naming for localized strings. For info in https://github.com/wordpress-mobile/WordPress-iOS/pull/20111

Reply:
Fixed! [9e98b61](https://github.com/wordpress-mobile/WordPress-iOS/pull/20135/commits/9e98b61d6d9ff412ce0c09825b9bb7e97097bccb)